### PR TITLE
Kernel no overwrite

### PR DESCRIPTION
**Pull request by JoseSantosAMD**
This PR removes kernel-vebose from profile. It now only shows up in analyze and database mode. Instead of overwriting names in existing csvs, it reads in the csvs to a dataframe and renames kernel names in those dataframes. In database mode these updated dataframes are used instead of the csvs,. They are uploaded with pymongo to upload to mongodb

### CODE REVIEW

File: src/utils/kernel_name_shortener.py [hunk 1]
```diff
@@ -125,18 +125,9 @@ def shorten_file(df, level):
...
-                    "profiling", "Skipping shortening on empty csv: %s" % str(fpath)
-                )
-
-        console_log("profiling", "Kernel_Name shortening complete.")
+        try:
+            modified_df = shorten_file(df, level)
+            console_log("profiling", "Kernel_Name shortening complete.")
+            return modified_df
+        except pd.errors.EmptyDataError:
+            logging.debug("[profiling] Skipping shortening on empty csv")
```
Comment (coleramos425) [inline src/utils/kernel_name_shortener.py:133]: Instead let's use something like this with our `console_debug()` helper
```suggestion
            console_debug("profiling", "Skipping shortening on empty csv")
```

File: src/utils/kernel_name_shortener.py [hunk 2]
```diff
@@ -23,18 +23,18 @@
...
 
 import os
-import glob
+import logging
 import re
 import subprocess
 import pandas as pd
 
-from utils.utils import console_error, console_debug, console_log
+from utils.utils import console_error, console_log
```
Comment (coleramos425) [inline src/utils/kernel_name_shortener.py:31]: Keep `console_debug()`
```suggestion
from utils.utils import console_error, console_log, console_debug
```

**Review by coleramos425 — CHANGES_REQUESTED** [review]
This PR will close an issue that a customer filed about kernel names being overridden ([#193](https://github.com/AMDResearch/omniperf/issues/193)) so could you also add a description to the PR explaining your patch and new behavior? Thanks.

**Comment by JoseSantosAMD** [comment]
This PR removes kernel-vebose from profile. It now only shows up in analyze and database mode. Instead of overwriting names in existing csvs, it reads in the csvs to a dataframe and renames kernel names in those dataframes. In database mode these updated dataframes are used instead of the csvs,. They are uploaded with pymongo to upload to mongodb

**Review by coleramos425 — APPROVED** [review]

**Comment by coleramos425** [comment]
Closes #193

**Pull request merged**